mono/metadata/class-internals.h

```diff
@@ -1067,28 +1068,64 @@ MONO_API gboolean
 mono_metadata_load_generic_param_constraints_checked (MonoImage *image, guint32 token,
 					      MonoGenericContainer *container, MonoError *error);
 
+// Tbis is the "real" function for registering JIT icalls. All others are one line wrappers that call it,
```

Comment:
Nit: typo: Tbis should be This

Reply:
There is another round of changes in a later PR, maybe this PR (depending on overall feedback). I'll fix it then, so to not upset the CI green and almost green status.